Add `--public-ignore` to ignore public metacomments (#5716)
verilator/verilator: docs/guide/exe_verilator.rst

   :code:`/*verilator public_flat_rd*/`
   metacomments.

.. option:: --public-off

   Ignore all
   :code:`/*verilator public* */`

>    metacomments. This is useful for optimized vpi builds, where a sim records reads/writes, then passes those with a .vlt file.

Reviewer: Add description of how works if give --public flag.

   :code:`/*verilator public_flat_rd*/`
   metacomments.

.. option:: --public-ignore

   Ignore all
   :code:`/*verilator public* */`

>    metacomments. This is useful for optimized vpi builds, where a sim records reads/writes, then passes those with a .vlt file.

Reviewer: vpi->VPI, sim->simulation.

Is below right?   We don't have an option to record read/writes - if that's a later pull update the docs then.

```suggestion
   metacomments. This is useful for speed-optimizing VPI builds where VPI is not being used.
```

>    Other control flags like :vlopt:`--public`, :vlopt:`--public-depth`, etc. will also work with this, as they are not metacomments.

Reviewer: ```suggestion
   This only affects metacomments; options such as :vlopt:`--public`, :vlopt:`--public-depth`, etc. work normally.
```
